deploy-board/deploy_board/templates/configs/new_capacity_adv.html

```diff
@@ -56,6 +56,7 @@ <h4 class="panel-title pull-left">Capacity</h4>
                         <div class="form-group"></div>
                         <div class="form-group">
                             <label class="deployToolTip control-label col-xs-2" data-toggle="tooltip" title="Customize user data">User data configs:</label>
+                            <a href="{{user_data_config_settings_wiki}}">wikiHelp</a>
```

Comment:
what if the env var is not set in non pinterest case? should we not show the wikihelp text?

Reply:
maybe use the "?" icon instead to be consistent with other ui element

Reply:
Thanks for pointing out, updated.